src/deadline_test_fixtures/fixtures.py

```diff
@@ -453,14 +456,39 @@ def worker_config(
             )
         file_mappings = [(resolved_whl_path, dest_path)]
 
-        LOG.info(f"The whl file will be copied to {dest_path} on the Worker environment")
-        worker_agent_requirement_specifier = dest_path
+        LOG.info(
+            f"The worker agent whl file will be copied to {dest_path} on the Worker environment"
+        )
+        pip_install_requirement_specifiers.append(dest_path)
     else:
         worker_agent_requirement_specifier = os.getenv(
             "WORKER_AGENT_REQUIREMENT_SPECIFIER",
             "deadline-cloud-worker-agent",
         )
         LOG.info(f"Using Worker agent package {worker_agent_requirement_specifier}")
+        pip_install_requirement_specifiers.append(worker_agent_requirement_specifier)
+
+    openjd_sessions_whl_path = os.getenv("OPENJD_SESSIONS_WHL_PATH")
+    if openjd_sessions_whl_path:
+        LOG.info(f"Using openjd-sessions agent whl file: {openjd_sessions_whl_path}")
+        resolved_whl_paths = glob.glob(openjd_sessions_whl_path)
+        assert (
+            len(resolved_whl_paths) == 1
+        ), f"Expected exactly one openjd-sessions whl path, but got {resolved_whl_paths} (from pattern {openjd_sessions_whl_path})"
+        resolved_whl_path = resolved_whl_paths[0]
+
+        if operating_system.name == "AL2023":
```

Comment:
```suggestion
        if operating_system.name.startswith("AL"):
```

Optional, but if we ever end up needing to support a new edition of AL then this becomes one less place to modify

Reply:
I went ahead and added this along with `OperatingSystem.is_amazon_linux()` and `OperatingSystem.is_windows()` methods to abstract these checks.

Reply:
Nice, even better. Thanks!